dohq_artifactory/admin.py

```diff
@@ -11,6 +12,19 @@ def rest_delay():
     time.sleep(0.5)
 
 
+def raise_errors(r):
+    try:
+        r.raise_for_status()
+    except requests.HTTPError as e:
+        if e.response.status_code >= 400:
+            logging.warning("Artifactory returned an error. Response text: {}".format(e.resonse.text))
+            raise ArtifactoryException
```

Comment:
May be we should add this extra-infromation about error to `ArtifactoryException`? 
Like this: `ArtifactoryException(e.response.text)`

Reply:
Fixed in latest commit.